dockstore-client/src/main/java/io/dockstore/common/FileProvisionUtil.java

```diff
@@ -69,27 +75,48 @@ private FileProvisionUtil() {
     }
 
     static boolean downloadFromVFS2(String path, Path targetFilePath, int threads) {
+
+        // An extension of the Http4sFileProvider class to allow for setting the cookie specification to something
+        // other than DEFAULT. This cannot be set through the fileSystemOptions argument.
+        class CustomHttp4sFileProvider extends Http4sFileProvider {
+            public HttpClientContext createHttpClientContext(final Http4FileSystemConfigBuilder builder,
+                                                             final GenericFileName rootName, final FileSystemOptions fileSystemOptions,
+                                                             final UserAuthenticationData authData) throws FileSystemException {
+
+                HttpClientContext def = super.createHttpClientContext(builder, rootName, fileSystemOptions, authData);
+                if (rootName.getHostName().equals("github.com")) {  // Github responds with a cookie header containing a
+                    // date in 4-digit year format. DEFAULT only allows 2-digit years; STANDARD allows 4-digits.
+                    // TODO: DEFAULT cookie spec may be outdated for more sites than Github
```

Comment:
This sentence sounds wrong, I think because of `TODO`, which implies you're going to change it, whereas we're not sure we should change it.

Reply:
Removed the TODO